web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -231,6 +231,53 @@
             var type,
               layer = layers[i];
             if (layer.group == "Background layers") {
+              var layerAttributionArray;
+              if (layer.extension && layer.extension.attribution) {
+                layerAttributionArray = [];
+                for (var a = 0; a < layer.extension.attribution.length; a++) {
+                  var attribution = layer.extension.attribution[a];
```

Comment:
```suggestion
              if (context.general.extension && context.general.extension.attribution) {
                layerAttributionArray = [];
                for (var a = 0; a < context.general.extension.attribution.length; a++) {
                  var attribution = context.general.extension.attribution[a];
```

The extension is defined in the `context/general` element not in the `layer` element, no?

Reply:
@josegar74 it is intentional for it to be applied at the layer.  
If you have multiple basemaps, then each basemap can have it's own attribution which it how it should be since the attributions are mostly credits to the people who made the basemap.

Reply:
@ianwallen, but the initial feedback for extensions in the XSD I provideid, the element is not at the layer level, but the general level

I see this other element at the layer level, but it is call `VendorExtension`: https://github.com/highsource/ogc-schemas/blob/18ce89928006eee08ed7e5a6ce6ea016436a444b/schemas/src/main/resources/ogc/owc/0.3.1/owsContext.xsd#L217

Please check to update the code to use this element, thanks.

Reply:
I have updated the code to use the VendorExtension instead of Extension